src/App.tsx

```diff
@@ -1,9 +1,87 @@
+import { Route, Routes, useLocation } from 'react-router-dom';
 import './App.scss';
+import { useEffect } from 'react';
+// import { Product } from './Components/ProductCards/Product';
+import { FavoriteContextProvider } from './FavoriteContext';
+import { HomePAge } from './Components/Pages/HomePage/HomePage';
+import { Header } from './Components/Header/Header';
+import { NotFoundPage } from './Components/Pages/NotFoundPage/NotFoundPage';
+import { PhonesPage } from './Components/Pages/PhonesPage/PhonesPage';
+import { PhonePreview } from './Components/Pages/PhonePreview';
+import { ContactsPage } from './Components/Pages/Contacts/Contacts';
+import { Favorite } from './Components/Pages/Favorite/Favorite';
+import { Basket } from './Components/Pages/Basket/Basket';
+import { Rights } from './Components/Pages/Rights/Rights';
+import { Footer } from './Components/Footer/Footer';
+// import { BlockBasket } from './Components/Pages/Basket/BlockBasket';
 
-const App = () => (
-  <div className="App">
-    <h1>React Phone Catalog</h1>
-  </div>
-);
+export const App = () => {
+  const location = useLocation();
 
-export default App;
+  useEffect(() => {
+    window.scrollTo(0, 0);
+  }, [location]);
+
+  return (
+    <>
```

Comment:
why do we need fragment here?

Reply:
I created this element before writing the provider, fixed it, thanks